### ISSUE TITLE
Screen is Blank

### ISSUE THREAD

**Issue by NoobtasticBeginner** [Unity-Technologies/FPSSample#20]
I opened the project however the scene is empty and the Project does not show the level. I am using the 2018 .3 bo7 btw

**Comment by parker-dean** [comment]
You will need to use the project tools to load any levels to test.

![sss](https://user-images.githubusercontent.com/43921262/47681974-97999b00-db90-11e8-9cd0-bbaab9927769.jpg)

**Comment by parker-dean** [comment]
You can build a full game..
![game44](https://user-images.githubusercontent.com/43921262/47682247-6b324e80-db91-11e8-9ab2-bac8b6784ec9.jpg)

**Comment by NoobtasticBeginner** [comment]
Ok thank you very much I will try it when I get back :)

**Comment by NoobtasticBeginner** [comment]
this is what showing but the levels aren't

**Comment by ParhamTaher** [comment]
I don't seem to have the FPS Sample menu option at all. Any ideas? I am using Unity 2018.2.11f1.

**Comment by NoobtasticBeginner** [comment]
After I cloned the project and you should open the project from the github
folder in the document if you already downloaded github

**Comment by Liven28** [comment]
> 
> 
> I don't seem to have the FPS Sample menu option at all. Any ideas? I am using Unity 2018.2.11f1.

I think FPS sample is only available with 2018.3 beta version

**Comment by ParhamTaher** [comment]
Yes you are right @Liven28 , I had to download the newer beta version!
